infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/OpenShiftInternalRuntime.java

```diff
@@ -200,6 +203,27 @@ private void prepareOpenShiftProject(String projectName) throws InfrastructureEx
         }
     }
 
+    private void prepareOpenShiftPVCs(OpenShiftEnvironment osEnv, String namespace) throws InfrastructureException {
```

Comment:
Looks like this part is not covered by tests. Are you going to add tests in this PR?